ui/src/app/kvm/views/KVMList/KVMListTable/_index.scss

```diff
@@ -0,0 +1,42 @@
+@mixin KVMListTable {
+  .kvm-list-table {
+    td,
+    th {
+      &:nth-child(1) {
```

Comment:
I've been trying to add comments about which columns these refer to so the next developer has a clue about what they're editing. E.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/settings/views/Dhcp/DhcpList/_index.scss

Reply:
Thanks @huwshimi! I've added those in

Reply:
If this is a common pattern to layout tables. Maybe we should add a data-title attr to each cell so we can target then heading names?